test/fixtures.rb

```diff
@@ -170,19 +170,52 @@ def call(params)
   end
 end
 
+class YMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    code, headers, body = @app.call(env)
+    [code, headers.merge!('Y-Middleware' => 'OK'), body]
+  end
+end
+
 class XMiddleware
-  def self.call(env)
-    env['X-Middleware'] = 'OK'
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    code, headers, body = @app.call(env)
+    [code, headers.merge!('X-Middleware' => 'OK'), body]
   end
 end
 
-class UseAction
-  include Lotus::Action
+module UseAction
+  class Index
+    include Lotus::Action
+    use XMiddleware
+
+    def call(params)
```

Comment:
What do you think if we add the following line and prove it from the test?

```ruby
def call(params)
  self.body = 'foo'
end
```

.. and ..

```ruby
response.body.must_equal ['foo']
```

:beetle: